.github/workflows/greetings.yml

```
@@ -0,0 +1,26 @@
+name: Greetings
+
+on: [pull_request_target, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: >
+          Thank you for contributing to {ecosystemdata} and opening your first issue
+          here! We are happy to have your contributions. We encourage you to
+          introduce yourself to the community on the [introduction thread in
+          our Discussions](https://github.com/orgs/NOAA-FIMS/discussions/801).
+        pr-message: >
+          Thank you for contributing to {ecosystemdata} and opening your first PR
+          here! We are happy to have your contributions. Please ensure that the
+          PR is made to the dev branch and let us know if you need any help!
```

```suggestion
          PR is made to the main branch and let us know if you need any help!
```
Right now, we are using the main branch for this repo rather than a dev branch.